priv/emqx.schema

```diff
@@ -1702,7 +1702,7 @@ end}.
   {datatype, {duration, ms}}
 ]}.
 
-{mapping, "listener.ssl.$name.enable_crl_cache", "emqx.listeners", [
+{mapping, "listener.ssl.$name.enable_crl_check", "emqx.listeners", [
```

Comment:
should we keep the old mapping for compatibility?

Reply:
This feature is not released yet, so it's safe to rename now.  :smiley_cat: